_includes/footer-content.html

```diff
@@ -16,7 +16,7 @@
         <li><a href="https://github.com/bazelbuild/bazel/issues">Issue Tracker</a></li>
         <li><a href="https://docs.bazel.build">Documentation</a></li>
         <li><a href="{{ site.main_site_url }}/faq.html">FAQ</a></li>
-        <li><a href="https://docs.bazel.build/support.html">Support Policy</a></li>
+        <li><a href="{{ site.main_site_url }}/support.html">Get Support</a></li>
```

Comment:
Can you keep "Support Policy"? This was on purpose. the footer already contains how to Get support. the page is only useful for the Policy.